src/lib/lint.test.ts

```diff
@@ -90,6 +99,60 @@ describe('parseGradeLevels()', () => {
   });
 });
 
+describe('parseMajors()', () => {
+  test('detects known majors', () => {
+    expect(
+      parseMajors('Accounting and fashion design majors may apply')
+    ).toEqual(['Accounting', 'Fashion Design']);
+  });
+  test('does not detect unknown major', () => {
+    expect(parseMajors('Welsh majors may apply')).toEqual([]);
```

Comment:
I'm curious if we should allow unknown majors since we allow `free solo`

Reply:
Don't we already do?